labgrid/driver/sshdriver.py

```diff
@@ -135,28 +135,41 @@ def _start_own_master_once(self, timeout):
                                         start_new_session=True)
 
         try:
+            version = subprocess.run(["ssh", "-V"], capture_output=True, text=True)
+            version = re.match(r"^OpenSSH_(\d+)\.(\d+)", version.stderr)
+            major, minor = map(int, version.groups())
+
+            if major < 8 or (major == 8 and minor < 5):
+                ssh_good_version = False
+            else:
+                ssh_good_version = True
```

Comment:
We already have `_scp_supports_explicit_sftp_mode()`. We should move the version check to a separate method and add a method `_ssh_supports_fork_with_stderr()`. Performing the version check once in `__attrs_post_init__()` would probably be best.

Reply:
done